holoviews/plotting/bokeh/element.py

```diff
@@ -1451,7 +1451,7 @@ def update_frame(self, key, ranges=None, element=None):
         elif element is not None:
             self.current_frame = element
             self.current_key = key
-        items = element.items() if element else []
```

Comment:
Was this broken for Python 3? How come this wasn't raised by the tests?

Reply:
No, not broken. I just changed it so that the tuples are guaranteed to be tuples ensuring that I don't have to call ``wrap_tuple`` all over the place.